gulp/atom.js

```diff
@@ -1,13 +1,47 @@
-var gulp = require( 'gulp' );
-var atom = require( 'gulp-atom' );
-
-gulp.task( 'atom', function () {
-	return atom( {
-		srcPath: './compile',
-		releasePath: './build',
-		cachePath: './cache',
-		version: 'v0.26.0',
-		rebuild: false,
-		platforms: [ 'darwin-x64' ]
-	} );
+var gulp = require('gulp');
+var atom = require('gulp-atom');
+var symlink = require('gulp-sym');
+var fs = require('fs');
+var path = require('path');
+var runSequence = require('run-sequence');
+
+var settings = {
+	srcPath: './compile',
+	releasePath: './build',
+	cachePath: './cache',
+	version: 'v0.26.0',
+	rebuild: false,
+	platforms: ['darwin-x64']
+};
+
+var platformPath = function(platform) {
+	return path.join(
+		settings.releasePath, 
+		settings.version,
+		platform,
+		'/Electron.app/Contents/Resources/app'
```

Comment:
Yes, it's a bug. Fixed it, should be alright now, just checked it by myself on windows.